user/getting-started.md

```diff
@@ -1,51 +1,84 @@
+- [Docker](https://www.docker.com/community-edition) (optional)
+
+### macOS
+
+- [Docker for Mac](https://www.docker.com/docker-mac)
+
+### Windows
+
+- [Docker for Windows](https://www.docker.com/docker-windows)
+
+## Running with Docker (recommended)
+
+The easiest way to run the server is using Docker. You can start it with the
+following command:
+
+```bash
+$ docker run --privileged -p 9432:9432 --name bblfsh bblfsh/server
 ```
 
-**Build driver + container**
+If everything worked, it should output something like this:
+
 ```
-# Ignore the 'no buildable Go source files' message
-go get -u github.com/bblfsh/python-driver
+time="2017-06-01T09:12:22Z" level=debug msg="binding to 0.0.0.0:9432" 
+time="2017-06-01T09:12:22Z" level=debug msg="initializing runtime at /tmp/bblfsh-runtime" 
+time="2017-06-01T09:12:22Z" level=debug msg="starting server" 
+time="2017-06-01T09:12:22Z" level=debug msg="registering gRPC service" 
+time="2017-06-01T09:12:22Z" level=info msg="starting gRPC server" 
+```
+
+The only mandatory flag is [`--privileged`](https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities).
+Babelfish Server uses containers itself to run language drivers, which currently
+requires it to run in privileged mode.
+
+Exposing the Babelfish port (`9432`) with `-p 9432:9432:` makes it easier to
+connect to it from outside the container.
 
-cd $GOPATH/src/github.com/bblfsh/python-driver
-go get -v -t ./...
-bblfsh-sdk prepare-build
-make build
+Now you can test that it works by submitting a file for parsing:
+ 
 ```
+$ echo "import foo" > sample.py
+$ docker run -v $(pwd):/work --link bblfsh bblfsh/server bblfsh client --address=bblfsh:9432 /work/sample.py
+```
+
+First request might timeout, since the server has to fetch the required driver
+before responding. If it does, just retry.
 
-In future, this will be replaced by `docker fetch` for the pre-build language driver containers
+### Running standalone
 
-## Run
+Running Babelfish standalone requires getting the `bblfsh` binary. Currently
+this requires a working setup of [Go](https://golang.org/). You can get it with
+the following command:
+
+```bash
+$ go get -u github.com/bblfsh/server/...
 ```
-# Get the hash of the docker container you just built
-docker images | grep python-driver
 
-# Replace the hash (the first 7 chars of the latest commit in the driver project)
-docker run -it --rm -v $GOPATH/src/github.com/bblfsh/python-driver:/code bblfsh/python-driver:dev-<commit[:7]>/opt/driver/bin/driver parse-uast --format=prettyjson /code/tests/sources/comprehension.py
+This will install the `bblfsh` binary into `$GOPATH/bin`.
 
-# Example with hash included that gets the UAST for the given `.py` file:
-docker run -it --rm -v $GOPATH/src/github.com/bblfsh/python-driver:/code bblfsh/python-driver:dev-955fc5d /opt/driver/bin/driver parse-uast --format=prettyjson /code/tests/sources/comprehension.py
+**[Linux only!]** Run the server:
```

Comment:
Why linux only? If it'll only work with linux it'd be nice to indicate it at the beginning of the section, and explain why is that and if it may (or not) change in the future.